crescent/typedefs.py

```diff
@@ -1,13 +1,24 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Awaitable, Callable, Protocol, Sequence
+from typing import Any, Awaitable, Callable, Dict, Protocol, Sequence, Union
 
-if TYPE_CHECKING:
-    from .context import Context
+from hikari import PartialChannel, Role, User
```

Comment:
What's the reason for removing this? If you're doing type aliases, you can just put them in quotes:
```py
HookCallbackT = Callable[["Context", CommandOptionsT], Awaitable[HookResult]]
```

Of course, this isn't really a huge deal.